src/hooks/useCds/index.js

```diff
@@ -82,6 +88,8 @@ const applyCds = async function(patientData, setOutput, setIsLoadingCdsData, isT
             cdsApplyEnd: Date.now(),
             timeRequestSent: new Date(),
             patientReference: patientReference,
+            patientInfo: patientInfo,
```

Comment:
Why don't we output `patientInfo` from DashboardLibrary in the AnalyticsLibrary.cql itself as opposed to adding it from the Dashboard here. This is what we do with DecisionAids.

Reply:
Having patientInfo as its own attribute in the message makes it slightly easier for analytics, and making the change here means there is no need to update and deploy the CQL.

Reply:
That's fine with me.